api-documentation.js

```diff
@@ -86,9 +94,34 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
     ${aware ? html`<raml-aware
       .scope="${aware}"
       @api-changed="${this._apiChanged}"></raml-aware>` : ''}
+    ${this._renderServerSelector()}
     ${this._renderView()}`;
   }
 
+  _renderServerSelector() {
+    const { amf, narrow, noServerSelector, selectedServerType, selectedServerValue, selectedType, serversCount, noCustomServer } = this;
+    
+    const isMethodOrEndpoint = selectedType && (selectedType === "method" || selectedType === "endpoint");
+    const moreThanOneServer = serversCount >= 2;
+
+    const shouldRenderSelector = !noServerSelector && narrow;
+    const shouldShowSelector =  isMethodOrEndpoint && moreThanOneServer;
```

Comment:
Please, don't make any computations in the render function. Use getters instead.